SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -100,31 +101,43 @@ bool USpatialGDKEditorSettings::IsAssemblyNameValid(const FString& Name)
 {
 	const FRegexPattern AssemblyPatternRegex(SpatialConstants::AssemblyPattern);
 	FRegexMatcher RegMatcher(AssemblyPatternRegex, Name);
-
-	return RegMatcher.FindNext();
+	bool valid = RegMatcher.FindNext();
```

Comment:
```suggestion
	bool bValidAssemblyName = RegMatcher.FindNext();
```

Reply:
Generally we use `UpperCamelCase` for our variable names. Booleans start with `b` though.